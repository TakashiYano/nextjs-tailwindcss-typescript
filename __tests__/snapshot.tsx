import renderer from "react-test-renderer";

import AboutPage from "../pages/about";
import IndexPage from "../pages/index";

it("renders index unchanged", () => {
  const tree = renderer.create(<IndexPage />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders about unchanged", () => {
  const tree = renderer.create(<AboutPage />).toJSON();
  expect(tree).toMatchSnapshot();
});