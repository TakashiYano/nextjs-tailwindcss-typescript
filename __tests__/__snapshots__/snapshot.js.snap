// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders about unchanged 1`] = `
<h2>
  About!
</h2>
`;

exports[`renders index unchanged 1`] = `
<div>
  <h2>
    Index
  </h2>
  <button
    className="p-2"
    onClick={[Function]}
  >
    Click me!
  </button>
</div>
`;